src/providers/krb5/krb5_child.c

```diff
@@ -3682,16 +3682,19 @@ static krb5_error_code privileged_krb5_setup(struct krb5_req *kr,
 
     if (!(offline ||
             (kr->fast_val == K5C_FAST_NEVER && kr->validate == false))) {
-        kerr = copy_keytab_into_memory(kr, kr->ctx, kr->keytab, &mem_keytab,
-                                       NULL);
-        if (kerr != 0) {
-            DEBUG(SSSDBG_OP_FAILURE, "copy_keytab_into_memory failed.\n");
-            return kerr;
+        /* A Keytab is not used if fast with anonymous pkinit is used (and validate is false)*/
+        if(kr->validate || kr->cli_opts->fast_use_anonymous_pkinit == false){
```

Comment:
Hi,

thank you for the fix. Would you mind to use `kr->validate == true`  to be more consistent with the other comparisons?

Additionally, please add `Resolves: https://github.com/SSSD/sssd/issues/6531` at the end of the commit message.

bye,
Sumit

Reply:
I took a little different approach to make my change more consistent than you recommended. I believe my new change is clearer. If you would prefer me to use your recommendation let me know and I can make the change.

I think I did the resolve comment correctly. Please let me know if I missed something.